mappings/net/minecraft/server/network/ServerPlayerEntity.mapping

```diff
@@ -16,6 +16,7 @@ CLASS vj net/minecraft/server/network/ServerPlayerEntity
 	FIELD cv cameraPosition Lfo;
 	FIELD cw containerSyncId I
 	FIELD d interactionManager Lvk;
+	FIELD f ping I
```

Comment:
Too vague.

Reply:
Other options I could come up with are `pingMilliseconds` (or shorter `pingMs`)

Reply:
Either would be better.

Reply:
Personally I think `ping` was enough. The same way we don't use `distanceBlocks` or `colorArgb`, we shouldn't use `pingMilliseconds`. The unit belongs in a javadoc, not the name.